src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue16522.cs

```diff
@@ -0,0 +1,23 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+    public class Issue16522 : _IssuesUITest
+    {
+		public Issue16522(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "[Android] Tabs briefly display wrong background color when navigating between flyout items";
+
+		[Test]
+		[Category(UITestCategories.Shell)]
+		public void ShellTabBarBackgroundColor()
```

Comment:
The snapshot for Catalyst is pending and available in the latest build. Could you commit the images?

Reply:
@jsuarezruiz  I’ve committed the snapshot for Catalyst. Please let me know if you have any further concerns.